eslint.config.js

```diff
@@ -305,4 +305,10 @@ export default tseslint.config(
         },
     },
     ...astroConfigs.recommended,
+    {
+        files: ["help-beta/src/components/ZulipNote.astro"],
+        rules: {
+            "import/unambiguous": "off",
+        },
+    },
```

Comment:
I don't know why this was complaining, I just turned it off.

Reply:
Can you post the error in #documentation somewhere? We may be able to defer fixing that later to not delay the cutover, but it probably does reflect something wrong with the code.